src/main/memorygame/app.cljs

```diff
@@ -18,67 +18,54 @@
  }))
 
 
-(defn tegn-felt [x]
-      [:div.col
-       [:div.flip-card
-        [:div.flip-card-inner
-         [:div.flip-card-front [:img {:src "images/halloween-background.png"}]]
-         [:div.flip-card-back [:img {:src (str "images/halloween-" x ".png")}]]]]])
+(defn kort [xs]
+      [:td {:id (str "felt-" (first xs))}
+       [:div.col
+         [:div.flip-card
+          [:div.flip-card-inner
+           [:div.flip-card-front [:img {:src "images/halloween-background.png"}]]
+           [:div.flip-card-back [:img {:src (str "images/halloween-" (last xs) ".png")}]]]]]])
 
 
 (defn generer-billedrækken [x] (shuffle (concat (range 1 (inc x)) (range 1 (inc x)))))
 
 
 (defn miniapp []
       (let [
-            billedrækken (zipmap (range 1 17) (shuffle (concat (range 1 9) (range 1 9))))
-            funktionen (fn [xs] [:td {:id (str "felt-" (first xs))} (tegn-felt (last xs))])
+            kortrækken (map kort (zipmap (range 1 17) (shuffle (concat (range 1 9) (range 1 9)))))
```

Comment:
I stedet for at have fire linier hvor vi kalder map, så tænkte jeg at det var bedre at kalde map så tidligt som muligt.